github-orgs/kubeflow/org.yaml

```diff
@@ -1011,6 +1011,7 @@ orgs:
             - aws-kf-ci-bot
             privacy: closed
             repos:
+              caff2-operator: write
               katib: write
```

Comment:
Do you want to remove the other repos? Since it appears to be choking on the other repos?